example/e2e_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"testing"
 
 	example "github.com/bergundy/protoc-gen-nexus-temporal/gen/example/v1"
+	"github.com/bergundy/protoc-gen-nexus-temporal/gen/example/v1/examplev1nexustemporal"
```

Comment:
I think this can get fairly long. Can we use just the last part of the package name? E.g. if it's `com.example` -> `examplenexustemporal` and `com.example.v1` -> `examplenexustemporal`? I'm worried the import name is becoming too long.